build/index.js

```diff
@@ -119,6 +121,7 @@ function injectExternalLinkClasses($) {
       return;
     }
     $a.addClass("external");
+    $a.attr("rel", ` ${$a.attr("rel") || ""} noopener`.trim());
```

Comment:
Since this code is quite condense, I'd suggest to add a comment, that we don't want to overwrite existing `rel` values.

Speaking of … what happens if you run this twice?
Would it become `rel="nofollow noopener noopener"`? :thinking: 

(Sorry for making the code even more complicated!)

Reply:
No but here you have to choose between getter and setter, you can't call a setter with a getter it's just bad. And then you might as well not use the dollars if you want.
 https://developer.mozilla.org/en-US/docs/Web/API/Element/setAttribute